### ISSUE TITLE
Not the maintained repository?

### ISSUE THREAD

**Issue by Lamm-a** [nselem/bigscape-corason#5]
It seems this is not the maintained repository for BiG-SCAPE but rather [here](https://git.wageningenur.nl/medema-group/BiG-SCAPE/-/tree/master) is.

If this is the case it might be worth putting that in the README?

**Comment by nselem** [comment]
Yes, Good Idea, 
At the end we see it was easier to get messages in github, but we can add a note to both README files.

**Comment by Lamm-a** [comment]
Yeh your gitlab does let let outsiders from your university make accounts I think?

**Comment by jorgecnavarrom** [comment]
No, that was a big problem indeed. We have migrated to https://github.com/medema-group/BiG-SCAPE, which is now the official repository.
